src/native/crt.c

```diff
@@ -90,22 +127,25 @@ static void s_alloc_tracker_track(struct alloc_tracker *tracker, void *ptr, size
     alloc->size = size;
     alloc->time = time(NULL);
 
-#if defined(ALLOC_TRACKING_ENABLED)
-    aws_byte_buf_init(&alloc->stacktrace, tracker->allocator, 1600);
-    void *stack_frames[10];
-    int stack_depth = backtrace(stack_frames, AWS_ARRAY_SIZE(stack_frames));
-    char **symbols = backtrace_symbols(stack_frames, stack_depth);
-    struct aws_byte_cursor newline = aws_byte_cursor_from_c_str("\n");
-    for (int idx = 2; idx < stack_depth && idx < 10; ++idx) {
-        if (idx > 2) {
-            aws_byte_buf_append(&alloc->stacktrace, &newline);
+#if defined(ALLOC_TRACE_AVAILABLE)
+    if (s_memory_tracing == 2) {
+        /* capture stack frames */
+        void *stack_frames[2 + ALLOC_TRACING_FRAMES];
```

Comment:
Can you pull this 2 out somewhere like a const? It's also used in that memcpy below and that spooks me

Reply:
I'm gonna replace this with the one from common where I've cleaned all this up and made it runtime configurable and unit tested and such.